doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -103,6 +103,9 @@ Matter
 
 * Added a page about :ref:`ug_matter_device_optimizing_memory`.
 * Fixed an IPC crash on nRF5340 when Zephyr's main thread takes a long time.
+* Disabled OpenThread shell by default in Matter over Thread samples.
+* Enabled :kconfig:option:CHIP_FACTORY_RESET_ERASE_NVS by default, even for builds without factory data support.
+  The firmware now erases all flash pages in non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
```

Comment:
```suggestion
  The firmware now erases all flash pages in the non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
```